executor/aggfuncs/func_count.go

```diff
@@ -49,7 +56,7 @@ func (e *countOriginal4Int) UpdatePartialResult(sctx sessionctx.Context, rowsInG
 	for _, row := range rowsInGroup {
 		_, isNull, err := e.args[0].EvalInt(sctx, row)
 		if err != nil {
-			return 0, err
+			return DefPartialResult4CountSize, err
```

Comment:
`DefPartialResult4CountSize` is the memory usage of `AllocPartialResult`,  `UpdatePartialResult` will not use more memory, `return 0, err` here